recipe_scrapers/gousto.py

```diff
@@ -5,7 +5,7 @@
 class Gousto(AbstractScraper):
 
     @classmethod
-    def host(self):
+    def host(cls):
```

Comment:
oh!!! `cls` is much better than `self` here and everywhere with `def host` in the project 😅  Thanks for catching this